lib/pbench/agent/tool_data_sink.py

```diff
@@ -1216,6 +1223,8 @@ def record_tms(self, tms):
         with (mdlog_name).open("w") as fp:
             mdlog.write(fp)
 
+        self.redis_server.set(tm_data_key, json.dumps(tms, sort_keys=True))
```

Comment:
FYI, I've proposed this change as a separate PR in https://github.com/distributed-system-analysis/pbench/pull/2742.

Reply:
PR #2742 was merged, so this code can be dropped entirely.